### PR TITLE
BXMSPROD-523: added extended email notification to PR jobs

### PR DESCRIPTION
**Pull request by mbiarnes**
extended eMail notification for PRs on master (once this is working and people are OK with this, this will be cherry-picked to other PRs and to 7.26.x)

### CODE REVIEW

File: job-dsls/jobs/pr_jobs.groovy [hunk 1]
```diff
@@ -280,6 +280,30 @@ for (repoConfig in REPO_CONFIGS) {
...
+
+            extendedEmail {
+                recipientList('$ghprbActualCommitAuthorEmail')
+                defaultSubject('$DEFAULT_SUBJECT')
+                defaultContent('$DEFAULT_CONTENT')
+                contentType('default')
+                triggers {
+                    failure{
+                        subject('PR build FAILED: $JOB_BASE_NAME #$ghprbPullId')
+                        content('$ghprbPullTitle \nPlease go to $BUILD_URL \n(IMPORTANT: you need have access to RedHat VPM to access this link) \n\n${BUILD_LOG, maxLines=1000} \n\n${FAILED_TESTS}')
```
Comment (mareknovotny) [inline job-dsls/jobs/pr_jobs.groovy]: @mbiarnes have you experimented with the numbers of lines? Maybe we can limit that to 500 or smaller number. But this is not really blocking right now, we can change it anytime ;)

File: job-dsls/jobs/pr_jobs.groovy [hunk 2]
```diff
@@ -280,6 +280,30 @@ for (repoConfig in REPO_CONFIGS) {
...
+                        sendTo {
+                            recipientList()
+                        }
+                    }
+                    unstable {
+                        subject('PR build UNSTABLE: $JOB_BASE_NAME #$ghprbPullId')
+                        content('$ghprbPullTitle \nPlease go to $BUILD_URL \n(IMPORTANT: you need have access to RedHat VPM to access this link) \n\n${BUILD_LOG, maxLines=1000} \n\n${FAILED_TESTS}')
```
Comment (mareknovotny) [inline job-dsls/jobs/pr_jobs.groovy]: @mbiarnes have you experimented with the numbers of lines? Maybe we can limit that to 500 or smaller number. But this is not really blocking right now, we can change it anytime ;)

File: job-dsls/jobs/pr_jobs.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy]: there is a maxLines=1000 defined. We can easily adjust this value to 500

File: job-dsls/jobs/pr_jobs.groovy (hunk 2 above)
Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy]: will change this to 500

**Comment by mbiarnes** [comment]
@mareknovotny chaned to 500 lines.
merge whenever you want

**Pull request merged**